### DISCUSSION TITLE
对比Baichuan7B

### DISCUSSION THREAD

**Discussion by lucasjinreal** [ymcui/Chinese-LLaMA-Alpaca-2#36] [General]
### 提交前必须检查以下项目

- [X] 请确保使用的是仓库最新代码（git pull），一些问题已被解决和修复。
- [X] 我已阅读[项目文档](https://github.com/ymcui/Chinese-LLaMA-Alpaca-2/wiki)和[FAQ章节](https://github.com/ymcui/Chinese-LLaMA-Alpaca-2/wiki/常见问题)并且已在Issue中对问题进行了搜索，没有找到相似问题和解决方案
- [X] 第三方插件问题：例如[llama.cpp](https://github.com/ggerganov/llama.cpp)、[text-generation-webui](https://github.com/oobabooga/text-generation-webui)等，同时建议到对应的项目中查找解决方案

### 问题类型

效果问题

### 基础模型

LLaMA-2-7B

### 操作系统

None

### 详细描述问题

可否对比一下Baichuan7B，看看相较于Baichuan，各方面差异如何

### 依赖情况（代码类问题务必提供）

```
# 请在此处粘贴依赖情况
```

### 运行日志或截图

```
# 请在此处粘贴运行日志
```

**Comment by ymcui** [comment]
@GXYM 你用的是哪个模型？明显模型不对，或者用法有误。你这个截图的输出基本就是胡说八道的情况了。

**Comment by GXYM** [comment]
> @GXYM 你用的是哪个模型？明显模型不对，或者用法有误。你这个截图的输出基本就是胡说八道的情况了。

模型下载对的，我用了别的gradio启动脚本，等我换成官方的gradio启动脚本在测测

**Comment by GXYM** [comment]
> @GXYM 你用的是哪个模型？明显模型不对，或者用法有误。你这个截图的输出基本就是胡说八道的情况了。

用的是这个模型[ziqingyang/chinese-llama-2-7b]看来就是模型很拉跨，不是我用了那个脚本导致的, 和百川没法比
<img width="1534" alt="企业微信截图_cc6a6273-13fc-437f-b270-81fc4e13d18c" src="https://github.com/ymcui/Chinese-LLaMA-Alpaca-2/assets/33718991/e45e6d2b-8ae3-4a28-b563-4f79c22dc37c">

**Comment by ymcui** [comment]
@GXYM 你指向的模型是LLaMA-2啊 😅  我们的指令模型是Alpaca-2。

**Comment by GXYM** [comment]
> @GXYM 你指向的模型是LLaMA-2啊 😅 我们的指令模型是Alpaca-2。

那我在测测ziqingyang/chinese-llama-2-7b

**Comment by Jieni05** [comment]
> > @GXYM 你指向的模型是LLaMA-2啊 😅 我们的指令模型是Alpaca-2。
> 
> 那我在测测ziqingyang/chinese-llama-2-7b

我觉得你应该测的是ziqingyang/chinese-alpaca-2-7b

**Comment by Jieni05** [comment]
> 我下的就是ziqingyang/chinese-llama-2-7b这个模型过， 吓得我赶紧检查了我下载的模型，是chinese-llama-2-7b， 不是chinese-

但是chinese-alpaca-2-7b才是指令模型..是指你应该去测那个

**Comment by GXYM** [comment]
> 


哦哦，那我再去测测那个指令模型

**Comment by GXYM** [comment]
> > 我下的就是ziqingyang/chinese-llama-2-7b这个模型过， 吓得我赶紧检查了我下载的模型，是chinese-llama-2-7b， 不是chinese-
> 
> 但是chinese-alpaca-2-7b才是指令模型..是指你应该去测那个

确实应该用指令模型来进行对话任务，chinese-alpaca-2-7b看起来就正常了 （多谢）
<img width="1468" alt="企业微信截图_f5d50e10-3e01-49c5-a00c-d386920bba6e" src="https://github.com/ymcui/Chinese-LLaMA-Alpaca-2/assets/33718991/8c78f706-8382-4b93-ae53-a978ece5dd9c">

**Comment by sun1092469590** [comment]
> > > 我下的就是ziqingyang/chinese-llama-2-7b这个模型过， 吓得我赶紧检查了我下载的模型，是chinese-llama-2-7b， 不是chinese-
> > 
> > 
> > 但是chinese-alpaca-2-7b才是指令模型..是指你应该去测那个
> 
> 确实应该用指令模型来进行对话任务，chinese-alpaca-2-7b看起来就正常了 （多谢） <img alt="企业微信截图_f5d50e10-3e01-49c5-a00c-d386920bba6e" width="1468" src="https://user-images.githubusercontent.com/33718991/257428419-8c78f706-8382-4b93-ae53-a978ece5dd9c.png">

和百川比怎么样？ 我也是准备测一下

**Comment by GXYM** [comment]
> > > > 我下的就是ziqingyang/chinese-llama-2-7b这个模型过， 吓得我赶紧检查了我下载的模型，是chinese-llama-2-7b， 不是chinese-
> > > 
> > > 
> > > 但是chinese-alpaca-2-7b才是指令模型..是指你应该去测那个
> > 
> > 
> > 确实应该用指令模型来进行对话任务，chinese-alpaca-2-7b看起来就正常了 （多谢） <img alt="企业微信截图_f5d50e10-3e01-49c5-a00c-d386920bba6e" width="1468" src="https://user-images.githubusercontent.com/33718991/257428419-8c78f706-8382-4b93-ae53-a978ece5dd9c.png">
> 
> 和百川比怎么样？ 我也是准备测一下

还是差了一点点， 以下是一些例子对比
百川：
<img width="965" alt="企业微信截图_a2b27685-ec38-4a46-a00d-8ca0e97ee592" src="https://github.com/ymcui/Chinese-LLaMA-Alpaca-2/assets/33718991/87282bc9-e983-4e3d-b28d-47a9b371b700">

chinese-alpaca-2-7b：

<img width="954" alt="企业微信截图_22a7b6ad-6379-4863-a394-22b5c9e36e50" src="https://github.com/ymcui/Chinese-LLaMA-Alpaca-2/assets/33718991/87917d0d-21bd-4a13-b959-6de6e72fc5fe">

**Comment by ymcui** [comment]
@GXYM  咱也不知道你是怎么用的，比如你写的`写一段关于洗发水的广告词`，我测试的输出是：
```
亲爱的朋友们，你是否曾经感到头发油腻、毛躁无力？是否尝试过各种不同的洗发水却依然无法改善这种情况呢？现在，我们向你推荐一款全新的洗发水——"清爽秀发"。
这款洗发水采用最先进的技术和天然植物提取物制成，能够彻底清洁头皮并去除油脂污垢，使头发柔顺、光滑且富有光泽。不仅如此，它还富含维生素B5和天然植物精华，能有效滋养和修复受损的发丝，让你的秀发更加健康亮丽。
清爽秀发不仅适用于油性或混合型头发，还可以改善干燥或敏感头皮问题。无论你是日常使用还是洗发后护理，都能感受到它的舒适感与效果。现在就让"清爽秀发"成为你每天的必备品吧！
```

**我建议是先评测模型使用的正确性，再进行效果对比。** 目录中的examples目录里有示例输出，如果你得不到类似的输出说明某些环节存在问题。

考虑到以上已经不属于Issue类问题，如有问题请发在讨论区（已转移本thread）。

**Reply by GXYM** [reply]
前面一列是问题，后面一列是回答；多次提问答案不同，是大模型的通病
